megalodon/src/entities/notification.ts

```diff
@@ -7,7 +7,7 @@ namespace Entity {
     created_at: string
     id: string
     status?: Status
-    emoji?: string
+    reaction?: Reaction
```

Comment:
Why did you change this parameter?

Reply:
As per https://git.joinfirefish.org/firefish/firefish/-/blob/develop/packages/megalodon/src/entities/notification.ts?ref_type=heads#L10 , the reaction from a notification should be a `Reaction` object representing the actual reaction rather than just a string

Reply:
But Pleroma provides only string as emoji in notifications: https://github.com/h3poteto/megalodon/blob/950ff623a3a0bdac46f07787a5256a6d200b391d/megalodon/src/pleroma/entities/notification.ts#L10
How to convert it?

Reply:
Ahh... I'll have to look into that, but Glitch, Firefish, and Akkoma provide full strings. Furthermore, Pleroma is adding custom emoji reactions in its next update, so I think they'll end up changing that.

Reply:
OK, please change this field after Pleroma adds custom emoji reactions.

Reply:
How should we handle the difference between pleroma-style and glitch-style reactions for now?

Reply:
OK, please keep `emoji` field even though you add `reaction` field. After Pleroma supports custom emoji reactions, let's integrate it into `reaction` field.